### PR TITLE
Login entities

### PR DESCRIPTION
**Pull request by Amaoto1992**
Creation of Login entities.

### CODE REVIEW

File: lib/core/entities/artist_entity/artist_entity.dart [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+import '../music_entity/music_entity.dart';
+part 'artist_entity.freezed.dart';
+
+@freezed
+class Artist with _$Artist {
+  factory Artist({
+    required String id,
+    required String name,
+    required String about,
+    required String? socialNetwork,
```
Comment (oscarshaitan) [inline lib/core/entities/artist_entity/artist_entity.dart]: lets move this to a list of strings for the moment could be multiple social networks, also probably later we want to pass this to an object or a map of Map<String,String> "social_network name", "url"

File: lib/core/entities/judge_entity/judge_entity.dart [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+part 'judge_entity.freezed.dart';
+part 'judge_entity.g.dart';
+
+@freezed
+class JudgeEntity with _$JudgeEntity {
+  factory JudgeEntity({
+    required String id,
+    required String name,
+    required String about,
+    required String? socialNetwork,
```
Comment (oscarshaitan) [inline lib/core/entities/judge_entity/judge_entity.dart]: same as previous comment

File: lib/core/entities/sponsor_entity/sponsor_entity.dart [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+part 'sponsor_entity.g.dart';
+part 'sponsor_entity.freezed.dart';
+
+@freezed
+class SponsorEntity with _$SponsorEntity {
+  factory SponsorEntity({
+    required String id,
+    required String name,
+    required String about,
+    required String? socialNetwork,
```
Comment (oscarshaitan) [inline lib/core/entities/sponsor_entity/sponsor_entity.dart]: same

File: lib/core/entities/team_member_entity/team_member_entity.dart [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+part 'team_member_entity.g.dart';
+part 'team_member_entity.freezed.dart';
+
+@freezed
+class TeamMemberEntity with _$TeamMemberEntity {
+  factory TeamMemberEntity({
+    required String id,
+    required String name,
+    required String about,
+    required String? socialNetwork,
```
Comment (oscarshaitan) [inline lib/core/entities/team_member_entity/team_member_entity.dart]: same

**Review by oscarshaitan — CHANGES_REQUESTED** [review]

File: lib/core/entities/artist_entity/artist_entity.dart (hunk 1 above)
Comment (Amaoto1992) [reply lib/core/entities/artist_entity/artist_entity.dart]: Fixed.

File: lib/core/entities/team_member_entity/team_member_entity.dart (hunk 4 above)
Comment (Amaoto1992) [reply lib/core/entities/team_member_entity/team_member_entity.dart]: Fixed.

File: lib/core/entities/sponsor_entity/sponsor_entity.dart (hunk 3 above)
Comment (Amaoto1992) [reply lib/core/entities/sponsor_entity/sponsor_entity.dart]: Fixed.

File: lib/core/entities/judge_entity/judge_entity.dart (hunk 2 above)
Comment (Amaoto1992) [reply lib/core/entities/judge_entity/judge_entity.dart]: Fixed.

Comment (Amaoto1992) [reply lib/core/entities/judge_entity/judge_entity.dart:12]: Fixed.

File: lib/core/entities/artist_entity/artist_entity.dart [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+class Artist with _$Artist {
+  factory Artist({
+    required String id,
+    required String name,
+    required String about,
+    required List<String?> socialNetwork,
+    required List<String> photos,
+    required List<Music> music,
+  }) = _Artist;
+}
```
Comment (oscarshaitan) [inline lib/core/entities/artist_entity/artist_entity.dart]: missed the fromJson just like the JudgeEntity

File: pubspec.yaml [hunk 6]
```diff
@@ -1,8 +1,14 @@
 name: fiura_ecosystem
 description: A new Flutter project.
+<<<<<<< HEAD
+
+publish_to:
+
+=======
```
Comment (oscarshaitan) [inline pubspec.yaml]: need to solve this conflicts first  this cant be here!

**Review by oscarshaitan — CHANGES_REQUESTED** [review]
on the yaml there is some conflicts you need to solve first
What a Conflict means is that someone else also worked on the same file so there is no clear way to solve that conflict.

How to solve this is by looking your changes and the other changes and see what combinations will complain with both needs 

the idea is not only accept yours and remove the others because this will bug the other person feature.


for example
you added the freezer model  but the other person added other library  the solution will be add both.


another example is both added the same lib but with different versions. Ideally, you will pick the last version but you need to make sure that ll the code is compiling

I can bet you that the code in this pr is not compiling due the yaml conflicts

File: lib/core/entities/artist_entity/artist_entity.dart (hunk 5 above)
Comment (Amaoto1992) [reply lib/core/entities/artist_entity/artist_entity.dart]: Both things are fixed.

File: pubspec.yaml (hunk 6 above)
Comment (Amaoto1992) [reply pubspec.yaml]: Fixed.

**Review by oscarshaitan — APPROVED** [review]

**Pull request merged**